web/pandas/pdeps/0010-required-pyarrow-dependency.md

```diff
@@ -1,12 +1,48 @@
 # PDEP-10: PyArrow as a required dependency for default string inference implementation
 
-- Created: 17 April 2023
-- Status: Accepted
+- Created: 17 April 2023 (updated May 8, 2024)
+- Status: Rejected
 - Discussion: [#52711](https://github.com/pandas-dev/pandas/pull/52711)
               [#52509](https://github.com/pandas-dev/pandas/issues/52509)
 - Author: [Matthew Roeschke](https://github.com/mroeschke)
           [Patrick Hoefler](https://github.com/phofl)
-- Revision: 1
+- Revision: 2
+
+# Note
+
+This PDEP was originally accepted on May 8, 2023. However, after reviewing feedback posted
+on the feedback issue [#54466](https://github.com/pandas-dev/pandas/issues/54466), we, the members of
+the core team, have not decided with moving forward with this PDEP for pandas 3.0.
+
+The primary reasons for rejecting this PDEP are twofold:
+
+1) Requiring pyarrow as a dependency causes installation problems.
+   - Pyarrow does not fit or has a hard time fitting in space-constrained environments 
+such as AWS Lambda and WASM, due to its large size of around ~40 MB for a compiled wheel
+(which is larger than pandas' own wheel sizes)
+   - Installation of pyarrow is not possible on some platforms. We provide support for some
+less widely used platforms such as Alpine Linux (and there is third party support for pandas in
+pyodide, a WASM distribution of pandas), both of which pyarrow does not provide wheels for.
+  
+   While both of these reasons are mentioned in the drawbacks section of this PDEP, at the time of the writing
+of the PDEP, we underestimated the impact this would have on users, and also downstream developers.
+
+2) Many of the benefits presented in this PDEP can be materialized even with payrrow as an optional dependency.
+   
+   For example, as detailed in PDEP-14, it is possible to create a new string data type with the same semantics
```

Comment:
PDEP 14 does not change performance or memory savings if you do not have pyarrow installed

Reply:
added a note in parentheses at the end of that sentence.

Reply:
Did you push this up? I don't see anything in parentheses.

The way I am interpreting this now is "we don't need/care for pyarrow strings because we have always had a string data type using Python strings" - is that correct?

Reply:
I updated the PDEP-15 text, and forgot to remove the PDEP-10 changes.

I've removed the PDEP-10 changes now.